src/components/BottomNav/index.tsx

```diff
@@ -13,15 +11,11 @@ const useStyles = makeStyles(() =>
       bottom: 0,
     }}));
 
-export const BottomNav = () => {
-  const [currentPage, setCurrentPage] = useState("/user/myGardens");
+export const BottomNav = ({currentPage, handlePageChange}) => {
+
```

Comment:
Add props types for this component,
`BottomNav: React.FC<{currentPage:... , handlePageChange:...}> => ...`